python/ray/tune/ray_trial_executor.py

```diff
@@ -437,6 +546,8 @@ def start_trial(self, trial, checkpoint=None, train=True) -> bool:
             time.sleep(2)
             error_msg = traceback.format_exc()
             self._stop_trial(trial, error=True, error_msg=error_msg)
+            # Return True here so we don't try to start a new trial
+            # in the main event loop
             return False
```

Comment:
nice contradiction you have going on here

Reply:
:-D

Reply:
But more seriously, is it supposed to be return True or return False?

Reply:
Oh right, it should return False (I fixed the bad semantics you pointed out last review)

Reply:
The comment was left over from last round